app/views/auth_sources/_auth_source_card.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="col-xs-12 col-sm-3 col-md-2">
+  <div class="card-pf card-pf-view card-pf-view-xs card-pf-view-select card-pf-view-single-select">
+    <div class="card-pf-body">
```

Comment:
Same as the comment earlier, you could take a look on the implementation of the Card  React component [here](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/statistics/ChartBox.js#L89)